### DISCUSSION TITLE
Metadata duplicated to incorrect field

### DISCUSSION THREAD

**Discussion by sharepointnewbie** [microsoft-search/pnp-modern-search#2701] [Q&A]
Hi Guys, hoping you can help out a newbie.

I've set up a document library, added some documents, created metadata columns, and setup pnp-modern-search with some refiners based on the meta data columns. Its works great, but for one weird issue.

The two columns I have are document type, and product type.

For Document Type predefined metadata selection I have for example DocTypeA / DocTypeB / DocTypeC
For product Type predefined metadata selection I have for example ProdA / ProdB / ProdC

If I do a search, when the applicable filters are show I see the following
**Document Type**
DocTypeA (10)
DocTypeB (10) 
DocTypeC (10)
ProdB (22) _no matter what the search, if the affected docs are in the results this appears under Document Type filter with the correct number of results as shown under product type_
**Product Type**
ProdA (50)
ProdB (22)
ProdC (50)

My problem applies to a select number of documents, all are word docs(from the same source) where most others are PDF's. The issue does not apply to ALL word docs though.

Why is this Product Type meta tag appearing under Document Type refiner? 

TIA

**Comment by wobba** [comment]
If ProdB  appears I would re-check how you set up the cp/mp mapping - and could there be a mapping at a site overriding a global mapping?

**Reply by sharepointnewbie** [reply]
I see where youre coming from, but that would surely result in all product b items being returned, but its only a selection of word docs (from the same 1 or 2 sources

its more like these particular docs have some (incorrect) metadata that i cant see.

**Reply by wobba** [reply]
If you look at the respone JSON on the query API call, if the value is there - then it's either a data or API error. If the value is not there, then it's a web part issue. Can you verify this?

**Reply by sharepointnewbie** [reply]
I'd be happy to, if you can explain how? I'm afraid that's totally over my head.

**Reply by wobba** [reply]
F12, look at the network tab for a query with `postquery` in it, then look at the JSON response :) If you can see the values coming back the way it's displayed, then an API issue. The web parts should render exactly what is returned, without any re-ordering logic.
